pkg/workload/rand/BUILD.bazel

```diff
@@ -21,4 +23,24 @@ go_library(
     ],
 )
 
+go_test(
+    name = "rand_test",
+    srcs = ["rand_test.go"],
+    args = ["-test.timeout=295s"],
```

Comment:
Bazel defaults to "medium" but it seems like this test doesn't need more than a few seconds, so maybe `size = "small"?